### ISSUE TITLE
Enforce eventdiagnosis rule: If have "Undetermined" at the event level, should have no other diagnoses at event level.

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/whispersservices#206]
Title of this issue comes from the business rule as written, but Neil explained that what effectively should happen is if an event is submitted (events serializer) or an event's diagnoses updated (eventdiagnosis created/updated via eventdiagnoses serializer) with an event diagnosis of undetermined but _also_ another diagnosis that is a 'real' one (not undetermined), the undetermined eventdiagnosis should be ignored and not saved. or least an error returned.

In summary, an event cannot have an undetermined diagnosis and have an actual diagnosis at the same time.
